src/app/store/accounts/blockchain/stacks/stacks-accounts.ts

```diff
@@ -108,6 +108,9 @@ export const stacksAccountState = atom<StacksAccount[] | undefined>(get => {
   return ledgerAccounts ?? softwareAccounts;
 });
 
+/**
```

Comment:
I'm curious what happens here. We shouldn't use it anymore but need to keep it in case someone is relying on it?

Reply:
Added inline comment